daemon/exec.go

```diff
@@ -262,6 +261,9 @@ func (d *Daemon) ContainerExecStart(name string, stdin io.ReadCloser, stdout io.
 		}
 		return nil
 	case err := <-execErr:
+		if err := <-attachErr; err != nil {
+			logrus.Warn(err)
```

Comment:
Really if exec didn't fail I think this should always be nil, it's just waiting for the attach to finish before continuing.